scico/linop/xray/_xray.py

```diff
@@ -193,3 +194,116 @@ def _calc_weights(x0, dx, nx, angle, y0):
     weights = jnp.minimum(distance_to_next, width) / width
 
     return inds, weights
+
+
+class Parallel3dProjector:
+    """General-purpose, 3D, parallel ray X-ray projector."""
+
+    def __init__(
+        self,
+        input_shape: Shape,
+        P: ArrayLike,
+        det_shape: Shape,
+    ):
+        r"""
+        Args:
+            input_shape: Shape of input image.
+            P: (num_angles, 2, 4) array of homogeneous projection matrices.
```

Comment:
Detailed documentation of the CT geometry, including an explanation of the projection matrices and a diagram of the projection geometry would be very helpful.

Reply:
First thing: actually document these.

Example of putting diagram in the docstring: https://scico.readthedocs.io/en/stable/_autosummary/scico.linop.xray.svmbir.html#module-scico.linop.xray.svmbir

Reply:
Added explanation: <TODO: link to docs>